make_plant_parts_eia.py

```diff
@@ -738,9 +729,17 @@ def slice_by_ownership(self, plant_gen_df):
                 'There should be more records labeled as total.')
         return plant_gen_df
 
-    def denorm_plant_gen(self, qual_records):
-        """Denromalize the plant_gen_df."""
-        og_len = len(self.plant_gen_df)
+    def denorm_plant_gen(self, plant_gen_df, qual_records):
+        """
+        Denromalize the plant_gen_df.
+
+        This method is meant to be run inside of `prep_plant_gen_df()`.
+
+        Args:
+            plant_gen_df (pandas.DataFrame): generator record table, which has
+                been augmented via 'slice_by_ownership()'
+        """
```

Comment:
I modified this method so that it takes `plant_gen_df` as a method instead of reading the cached version so that it was standardized.